umm-lib/src/cmr/umm/iso_smap/granule/orbit.clj

```diff
@@ -50,40 +51,95 @@
             start-lat (convert-direction start-direction)
             end-lat (convert-direction end-direction))))
 
+(defn- build-ocsd-string
+  "Builds ISO SMAP orbit calculated spatial domain string using umm values."
+  [ocsd]
+  (let [{:keys [orbital-model-name orbit-number start-orbit-number stop-orbit-number
+                equator-crossing-longitude equator-crossing-date-time]} ocsd
+        OrbitalModelName (when orbital-model-name
+                           (str "OrbitalModelName: " orbital-model-name))
+        OrbitNumber (when orbit-number
+                      (str "OrbitNumber: " orbit-number))
+        StartOrbitNumber (when start-orbit-number
+                           (str "BeginOrbitNumber: " start-orbit-number))
+        StopOrbitNumber (when stop-orbit-number
+                          (str "EndOrbitNumber: " stop-orbit-number))
+        EquatorCrossingLongitude (when equator-crossing-longitude
+                                   (str "EquatorCrossingLongitude: " equator-crossing-longitude))
+        EquatorCrossingDateTime (when equator-crossing-date-time
+                                  (str "EquatorCrossingDateTime: " equator-crossing-date-time))]
+    (str OrbitalModelName OrbitNumber StartOrbitNumber StopOrbitNumber
```

Comment:
Should have " " in between the fields to make it more readable for humans.

Reply:
Done.